.github/workflows/pull-request.yaml

```diff
@@ -36,6 +36,13 @@ jobs:
   check_commited_files:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - name: Install Golang 1.20
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.20'
```

Comment:
Is it possible to fetch that version from `go.mod`?

Reply:
I'm not sure. If yes, we should first do it in `main` branch.

Reply:
We do

https://github.com/kubevirt/ssp-operator/blob/5abe053dec0f255d6c59ebef3407ddb03da529f8/.github/workflows/pull-request.yaml#L41

Reply:
Right. Sorry I forgot about that.
I will update this PR.